Sources/AppStoreConnectCLI/Commands/BundleIds/HTTPClient+BundleId.swift

```diff
@@ -17,9 +17,7 @@ extension HTTPClient {
         }
     }
 
-    /// Find the opaque internal identifier for a bundle ID matching `identifier`.
-    ///
-    ///  not the reverse-DNS bundleId identifier. Use this for reading, modifying and deleting bundleId resources.
+    /// Find the opaque internal identifier for a bundle ID matching `identifier`.  Use this for reading, modifying and deleting BundleId resources.
```

Comment:
Should we standardize on 'internal identifier' vs 'resource identifier'? Apple also seems to use both in their documentation.

Maybe we can just add 'resource':

```suggestion
    /// Find the opaque internal resource identifier for a bundle ID matching `identifier`.  Use this for reading, modifying and deleting BundleId resources.
```